{
  "cve": "CVE-2019-8325",
  "mitre": {
    "cpes": [],
    "created": "2019-06-17T18:59:57+00:00",
    "description": "An issue was discovered in RubyGems 2.6 and later through 3.0.2. Since Gem::CommandManager#run calls alert_error without escaping, escape sequence injection is possible. (There are many ways to cause an error.)",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2019/8xxx/CVE-2019-8325.json",
    "references": [
      "http://lists.opensuse.org/opensuse-security-announce/2019-07/msg00036.html",
      "https://hackerone.com/reports/317353",
      "https://lists.debian.org/debian-lts-announce/2020/08/msg00027.html"
    ],
    "title": null,
    "updated": "2024-08-04T21:17:30.566000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:rubygems:rubygems:*:*:*:*:*:*:*:*",
      "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*",
      "cpe:2.3:o:opensuse:leap:15.0:*:*:*:*:*:*:*",
      "cpe:2.3:o:opensuse:leap:15.1:*:*:*:*:*:*:*"
    ],
    "created": "2019-06-17T19:15:11.890000+00:00",
    "description": "An issue was discovered in RubyGems 2.6 and later through 3.0.2. Since Gem::CommandManager#run calls alert_error without escaping, escape sequence injection is possible. (There are many ways to cause an error.)",
    "metrics": {
      "cvssV2_0": {
        "score": 5.0,
        "vector": "AV:N/AC:L/Au:N/C:N/I:P/A:N"
      },
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 7.5,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2019/CVE-2019-8325.json",
    "references": [
      "http://lists.opensuse.org/opensuse-security-announce/2019-07/msg00036.html",
      "https://hackerone.com/reports/317353",
      "https://lists.debian.org/debian-lts-announce/2020/08/msg00027.html"
    ],
    "title": null,
    "updated": "2020-08-19T19:01:44.597000+00:00",
    "vendors": [
      "debian",
      "debian$PRODUCT$debian_linux",
      "opensuse",
      "opensuse$PRODUCT$leap",
      "rubygems",
      "rubygems$PRODUCT$rubygems"
    ],
    "weaknesses": [
      "CWE-74"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:/a:redhat:cloudforms_managementengine:5.10::el7",
        "cpe:/a:redhat:enterprise_linux:8",
        "cpe:/a:redhat:rhel_software_collections:3::el6",
        "cpe:/a:redhat:rhel_software_collections:3::el7",
        "cpe:/o:redhat:enterprise_linux:7",
        "cpe:/o:redhat:rhel_aus:7.4",
        "cpe:/o:redhat:rhel_e4s:7.4",
        "cpe:/o:redhat:rhel_tus:7.4",
        "cpe:2.3:a:rubygems:rubygems:*:*:*:*:*:*:*:*",
        "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*",
        "cpe:2.3:o:opensuse:leap:15.0:*:*:*:*:*:*:*",
        "cpe:2.3:o:opensuse:leap:15.1:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "created": {
      "data": "2019-03-05T00:00:00+00:00",
      "provider": "redhat"
    },
    "description": {
      "data": "An issue was discovered in RubyGems 2.6 and later through 3.0.2. Since Gem::CommandManager#run calls alert_error without escaping, escape sequence injection is possible. (There are many ways to cause an error.)",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 5.0,
          "vector": "AV:N/AC:L/Au:N/C:N/I:P/A:N"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {
          "score": 5.3,
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
        },
        "provider": "redhat"
      },
      "cvssV3_1": {
        "data": {
          "score": 7.5,
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Low",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "http://lists.opensuse.org/opensuse-security-announce/2019-07/msg00036.html",
        "https://hackerone.com/reports/317353",
        "https://lists.debian.org/debian-lts-announce/2020/08/msg00027.html",
        "https://nvd.nist.gov/vuln/detail/CVE-2019-8325",
        "https://www.cve.org/CVERecord?id=CVE-2019-8325"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    },
    "title": {
      "data": "rubygems: Escape sequence injection vulnerability in errors",
      "provider": "nvd"
    },
    "updated": {
      "data": "2024-08-04T21:17:30.566000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "debian",
        "debian$PRODUCT$debian_linux",
        "opensuse",
        "opensuse$PRODUCT$leap",
        "redhat",
        "redhat$PRODUCT$cloudforms_managementengine",
        "redhat$PRODUCT$enterprise_linux",
        "redhat$PRODUCT$rhel_aus",
        "redhat$PRODUCT$rhel_e4s",
        "redhat$PRODUCT$rhel_software_collections",
        "redhat$PRODUCT$rhel_tus",
        "rubygems",
        "rubygems$PRODUCT$rubygems"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-74",
        "CWE-88"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [
      "cpe:/a:redhat:cloudforms_managementengine:5.10::el7",
      "cpe:/a:redhat:enterprise_linux:8",
      "cpe:/a:redhat:rhel_software_collections:3::el6",
      "cpe:/a:redhat:rhel_software_collections:3::el7",
      "cpe:/o:redhat:enterprise_linux:7",
      "cpe:/o:redhat:rhel_aus:7.4",
      "cpe:/o:redhat:rhel_e4s:7.4",
      "cpe:/o:redhat:rhel_tus:7.4"
    ],
    "created": "2019-03-05T00:00:00+00:00",
    "description": "An issue was discovered in RubyGems 2.6 and later through 3.0.2. Since Gem::CommandManager#run calls alert_error without escaping, escape sequence injection is possible. (There are many ways to cause an error.)",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {
        "score": 5.3,
        "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
      },
      "cvssV3_1": {},
      "threat_severity": "Low"
    },
    "redhat_repo_path": "2019/CVE-2019-8325.json",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2019-8325",
      "https://www.cve.org/CVERecord?id=CVE-2019-8325"
    ],
    "title": "rubygems: Escape sequence injection vulnerability in errors",
    "updated": "2019-03-05T00:00:00+00:00",
    "vendors": [
      "redhat",
      "redhat$PRODUCT$cloudforms_managementengine",
      "redhat$PRODUCT$enterprise_linux",
      "redhat$PRODUCT$rhel_aus",
      "redhat$PRODUCT$rhel_e4s",
      "redhat$PRODUCT$rhel_software_collections",
      "redhat$PRODUCT$rhel_tus"
    ],
    "weaknesses": [
      "CWE-88"
    ]
  }
}